docs/analyses/food_supply_of_world_regions/food_supply_of_world_regions.ipynb

```diff
@@ -0,0 +1,2055 @@
+                "* Changes in historical regions: the dissolution of the USSR significantly altered the geographical areas classified as Europe and Asia.\n",
+                "\n",
+                "\n",
+                "The aim is to ensure internal consistency across time, allowing reliable visualization of long-term trends rather than precise measurement of absolute levels.\n",
+                "By the end of this notebook, we will have a data table with population, food supply (in kilocalories), and agricultural land use (in hectares), for every continent, fulfilling the following conditions:\n",
+                "* Regions refer to the same set of countries each year and for each indicator.\n",
+                "* The data does not show abrupt, spurious jumps."
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "id": "4530449d",
+            "metadata": {},
+            "source": [
+                "## Initial setup\n",
+                "### Install and import requirements\n",
+                "\n",
+                "To be able to run the code, there are some libraries you may need to install."
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "id": "01d97fa0",
+            "metadata": {
+                "tags": [
+                    "collapsed"
+                ]
+            },
+            "outputs": [],
+            "source": [
+                "# If running on Colab, install all dependencies.\n",
+                "%pip install --upgrade owid-catalog plotly > /dev/null 2>&1"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "id": "6566cb9d",
+            "metadata": {},
+            "source": [
+                "Import necessary libraries."
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": 2,
+            "id": "7a6beeed",
+            "metadata": {
+                "tags": [
+                    "collapsed"
+                ]
+            },
+            "outputs": [],
+            "source": [
+                "import json\n",
+                "import plotly.express as px\n",
+                "from owid.catalog import find, processing as pr"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "id": "8f959d1e",
+            "metadata": {},
+            "source": [
+                "Set common variables."
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": 3,
+            "id": "e218a319",
+            "metadata": {
+                "tags": [
+                    "collapsed"
+                ]
+            },
+            "outputs": [],
+            "source": [
+                "# Other variables that should only be modified by OWID in case there is a data update.\n",
```

Comment:
Again, I'd elaborate a bit on this, I think the note is currently a bit confusing. If you are telling users not to change these, I'd say something like: "These variables ensure the analysis is run correctly and with the most recent data, there is no need to change them".